gsoc/cms_toolbars.py

```diff
@@ -51,6 +51,11 @@ def add_admin_menu(self):
 
         user = getattr(self.request, "user", None)
 
+        if user:
+            user_profile = UserProfile.objects.filter(user=user)[0]
```

Comment:
cant just use the first profile, have to look for the userprofile of student for this current gsoc year